aws-py-oidc-provider/__main__.py

```diff
@@ -0,0 +1,70 @@
+import pulumi
+from pulumi_aws import iam
+import requests
+import subprocess
+import OpenSSL
+import json
+
+audience = "" # Provide the name of your Pulumi Organization
```

Comment:
Use `config.require()`: https://www.pulumi.com/docs/reference/pkg/python/pulumi/#pulumi.Config.require

Reply:
Is there a built-in way to retrieve the org associated with the stack? Or do users have to explicitly configure that?

Reply:
There's a built-in way to retrieve the organization, project, and stack name. I've done this in Go many times, but in Python I think you need to use `pulumi.get_organization`. I'll try to find a link.